content/applications/general/developer_mode.rst

```diff
@@ -4,76 +4,80 @@
+Debug` extension. Once the extension is installed, a new icon will be shown on the browser's
+toolbar.
 
-.. image:: developer_mode/monkey.png
+For the *Odoo Debug* extension, a single click enables a normal version of the mode, while a
+double click enables it with assets. To deactivate it, use a single click.
+
+.. image:: developer_mode/developer-mode-monkey.png
    :align: center
-   :alt: View of odoo’s debug icon in a chrome’s toolbar
+   :alt: View of Odoo's debug icon in a Google Chrome toolbar.
 
 Activate through the command palette
 ====================================
 
-The command palette tool has a command to activate the debug mode: open it with
-the keyboard shortcut `ctrl+k`, then type `debug`: a command will show up to
-activate the debug mode. 
+In Odoo the command palette tool has a command to activate the debug mode. First, open the command
+palette tool with the keyboard shortcut `ctrl+k` and then type `debug`. A command will show up to
+activate the debug mode.
 
-.. image:: developer_mode/command_palette.png
+.. image:: developer_mode/command-palette.png
    :align: center
-   :alt: Command palette with debug command
+   :alt: Command palette with debug command.
 
 Activate through the URL
 ========================
 
-In the URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the
-debug mode, add `?debug=0` instead.
+In the URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the debug mode, add
+`?debug=0` instead.
 
 .. image:: developer_mode/url.png
    :align: center
-   :alt: Overview of an url with the debug mode command added in Odoo
+   :alt: Overview of a URL with the debug mode command added.
 
 .. tip::
-   Additional modes are available for developers: `?debug=assets` enables the
-   :ref:`assets mode <frontend/framework/assets_debug_mode>`, and `?debug=tests` enables
-   the :ref:`tests mode <frontend/framework/tests_debug_mode>`.
+   Additional modes are available for developers: `?debug=assets` enables the :ref:`assets mode
+   <frontend/framework/assets_debug_mode>`, and `?debug=tests` enables the :ref:`tests mode
+   <frontend/framework/tests_debug_mode>`.
 
 Locate the mode tools
 =====================
 
-The Developer mode tools can be accessed from the *Open Developer Tools* button,
-located on the header of your pages. This menu contains additional tools that
-are useful to understand or edit technical data, such as the views or the actions.
-It contains some useful menu items such as:
+Once developer mode is activated, the developer mode tools can be accessed from the *Open Developer
+Tools* button, located on the header of the Odoo database.
+
+.. image:: developer_mode/button-location.png
+   :align: center
+   :alt: Overview of a console page and the debug icon being shown in Odoo.
+
+This menu contains additional tools that are useful to understand or edit technical data, such as
+the views or the actions. It contains some useful menu items such as:
```

Comment:
- Delete `the` from `the actions`
- Add `of a page` after `actions`
- Replace `It contains` with `On a page that has filters, actions, and view options, the developer mode tools contain`